proto/src/determined/experiment/v1/searcher.proto

```diff
@@ -41,13 +41,13 @@ message TrialProgress {
 message ValidationCompleted {
   option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
     json_schema: {
-      required: [ "request_id", "metric", "validate_after_length" ]
+      required: [ "request_id", "metrics", "validate_after_length" ]
     }
   };
   // UUID identifying the trial to the searcher.
   string request_id = 1;
-  // Value of the validation metric used to direct the search.
-  double metric = 2;
+  // Values of the validation metrics used to direct the search.
+  google.protobuf.Value metrics = 2;
```

Comment:
slightly incongruent with the name we use in the python.

This is such a tiny nit though, I don't care much.